### PR TITLE
p2-rctl: accept RC labels to be applied to the RC

### PR DESCRIPTION
**Pull request by petertseng**

### CODE REVIEW

File: bin/p2-rctl/main.go [hunk 1]
```diff
@@ -104,7 +105,6 @@ func main() {
 	}
 
 	httpClient := cleanhttp.DefaultClient()
-	opts.Client = httpClient
```
Comment (petertseng) [inline bin/p2-rctl/main.go]: I'll happily take alternatives that allow me to use this in `opts.Client` (just need to be able to set headers based on our command line flags), but I don't see any in  https://github.com/hashicorp/go-cleanhttp/blob/master/cleanhttp.go

File: bin/p2-rctl/main.go [hunk 2]
```diff
@@ -53,6 +53,7 @@ var (
 	createManifest  = cmdCreate.Flag("manifest", "manifest file to use for this replication controller").Short('m').Required().String()
 	createNodeSel   = cmdCreate.Flag("node-selector", "node selector that this replication controller should target").Short('n').Required().String()
 	createPodLabels = cmdCreate.Flag("pod-label", "a pod label, in LABEL=VALUE form, to add to this replication controller. Can be specified multiple times.").Short('p').StringMap()
+	createRCLabels  = cmdCreate.Flag("rc-label", "an RC label, in LABEL=VALUE form, to be applied to to this replication controller. Can be specified multiple times.").Short('r').StringMap()
```
Comment (petertseng) [inline bin/p2-rctl/main.go]: to to!

File: bin/p2-rctl-server/main.go [hunk 3]
```diff
@@ -71,7 +71,6 @@ func main() {
 
 	// Initialize the myriad of different storage components
 	httpClient := cleanhttp.DefaultClient()
```
Comment (anthonybishopric) [inline bin/p2-rctl-server/main.go]: This variable appears to be unused now. Is there a compiler warning?

Comment (anthonybishopric) [inline bin/p2-rctl-server/main.go]: Nevermind, chrome highlighting fail.

**Comment by anthonybishopric** [comment]
Good for now, :+1:

**Comment by petertseng** [comment]
text change only

```diff
$ git diff 2c1e2a9 efbe2d6
diff --git a/bin/p2-rctl/main.go b/bin/p2-rctl/main.go
index a2b94d5..f776bc1 100644
--- a/bin/p2-rctl/main.go
+++ b/bin/p2-rctl/main.go
@@ -53,7 +53,7 @@ var (
 	createManifest  = cmdCreate.Flag("manifest", "manifest file to use for this replication controller").Short('m').Required().String()
 	createNodeSel   = cmdCreate.Flag("node-selector", "node selector that this replication controller should target").Short('n').Required().String()
 	createPodLabels = cmdCreate.Flag("pod-label", "a pod label, in LABEL=VALUE form, to add to this replication controller. Can be specified multiple times.").Short('p').StringMap()
-	createRCLabels  = cmdCreate.Flag("rc-label", "an RC label, in LABEL=VALUE form, to be applied to to this replication controller. Can be specified multiple times.").Short('r').StringMap()
+	createRCLabels  = cmdCreate.Flag("rc-label", "an RC label, in LABEL=VALUE form, to be applied to this replication controller. Can be specified multiple times.").Short('r').StringMap()
 
 	cmdDelete   = kingpin.Command(CMD_DELETE, "Delete a replication controller")
 	deleteID    = cmdDelete.Arg("id", "replication controller uuid to delete").Required().String()

```

**Pull request merged**